### DISCUSSION TITLE
What is the terminal in README?

### DISCUSSION THREAD

**Discussion by jqhr** [willothy/flatten.nvim#31] [Q&A] [answered]
It  looks beautiful. I want to install it too.

**Comment by willothy** [comment, marked as answer]
> It  looks beautiful. I want to install it too.

Windows terminal, on WSL. Ended up switching to WezTerm a few days ago though.

**Comment by OmiChrome** [comment]
@willothy  im using window terminal , but how do i get transparent look , as u got in neovim ? 

are u using [plugin?](https://github.com/xiyaowong/transparent.nvim)

**Reply by willothy** [reply]
You can set transparency in highlight groups with the `blend` property, many color schemes support transparency.

**Reply by OmiChrome** [reply]
i dont get it , please provide me some example or sources!

**Reply by willothy** [reply]
https://neovim.io/doc/user/syntax.html#%3Ahi-link
